crates/compiler/test_syntax/src/test_helpers.rs

```diff
@@ -116,6 +133,81 @@ impl<'a> Output<'a> {
             Output::Full { .. } => format!("{self:#?}\n"),
         }
     }
+
+    pub fn canonicalize(&self, arena: &Bump, src: &str) {
+        set_panic_not_exit(true); // can has a bunch of internal_error! calls
+
+        match self {
+            Output::Header(_) => {}
+            Output::ModuleDefs(_) => {
+                // TODO: canonicalize module defs
```

Comment:
TODO

Reply:
Yep; my plan is to leave this for later. Expr's are the majority of our syntax tests, so just doing the expr case for now gives us quite a lot of coverage already.